quinn/src/platform/unix.rs

```diff
@@ -267,6 +268,11 @@ impl super::UdpExt for UdpSocket {
     }
 }
 
+/// Returns the platforms UDP socket capabilities
+pub fn caps() -> super::UdpCapabilities {
```

Comment:
Let's just import `UdpCapabilities` at the module level? All the `super::` prefixes get pretty tedious.

Reply:
sure. Do you prefer relative `super::` imports for that or absolute `crate::` imports?

Reply:
I think `super` makes sense in this case.